pkg/virt-operator/resource/apply/reconcile.go

```diff
@@ -336,21 +337,23 @@ func (r *Reconciler) createDummyWebhookValidator() error {
 			continue
 		}
 		path := fmt.Sprintf("/fake-path/%s", crd.Name)
-		webhooks = append(webhooks, admissionregistrationv1beta1.ValidatingWebhook{
-			Name:          fmt.Sprintf("%s-tmp-validator", crd.Name),
-			FailurePolicy: &failurePolicy,
-			Rules: []admissionregistrationv1beta1.RuleWithOperations{{
-				Operations: []admissionregistrationv1beta1.OperationType{
-					admissionregistrationv1beta1.Create,
+		webhooks = append(webhooks, admissionregistrationv1.ValidatingWebhook{
+			Name:                    fmt.Sprintf("%s-tmp-validator", crd.Name),
+			AdmissionReviewVersions: []string{"v1", "v1beta1"},
+			SideEffects:             &sideEffectNone,
```

Comment:
Non-trivial changes spotted

Reply:
Yes, the `AdmissionReviewVersions` is now mandatory with v1 `AdmissionReview` but I think that's making the downgrade test fail, I will have to fix that.

Reply:
> Yes, the `AdmissionReviewVersions` field is now mandatory with v1 `AdmissionReview` but I think that's making the downgrade test fail, I will have to fix that.

You mean the tests that fail on the 1.18 and 1.19 jobs?
Do you see it fail locally as well?